packages/editor/src/components/table-of-contents/panel.js

```diff
@@ -5,13 +5,22 @@ import { Fragment } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { withSelect } from '@wordpress/data';
 
+import { delay } from 'lodash';
```

Comment:
Unnecessary / inconsistent newline prior to this one should be removed.

Reply:
👍

Reply:
moved to top of file in https://github.com/WordPress/gutenberg/pull/10815/commits/44d6d40905e60fb432b77954e266e87a37b93830